src/test/java/emissary/core/channels/BufferedChannelFactoryTest.java

```diff
@@ -0,0 +1,28 @@
+package emissary.core.channels;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class BufferedChannelFactoryTest {
+    @Test
+    void testCache() throws IOException {
```

Comment:
Suggest adding a second test for which the buffer size is larger than the channel size.,

Reply:
Test updated.